packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/origin-name-02-util.output.js

```diff
@@ -0,0 +1,7 @@
+import { cookies } from 'next/headers'
+
+export default function Foo(): string {
+  const name = /* Next.js Dynamic Async API Codemod: Manually await this call, if it's a Server Component */
+  cookies().get('name')
```

Comment:
Why do we not typecast here?

Reply:
I assumed because this is a JS file... which means we should probably remove the `: string` on line 3.